unlock-app/src/components/content/CheckoutContent.tsx

```diff
@@ -40,14 +44,23 @@ interface CheckoutContentProps {
   errors: UnlockError[]
 }
 
-export const CheckoutContent = ({
+export const CheckoutContent = (props: CheckoutContentProps) => {
+  return (
+    <BrowserOnly>
```

Comment:
Needed to make all of this very ugly for the time being so that the server side rendering wouldn't break anything.

Reply:
Yeah I think there is no server side rendering which would make such sense here for now anyway.